src/internal-types.ts

```diff
@@ -120,6 +120,7 @@ export type AgentParameters = {
 export type AgentLogoutOptions = {
   customScheme?: string;
   federated?: boolean;
+  redirectUrl?: string;
```

Comment:
Would it make sense to name this returnTo when exposing to the user?

There is no such thing as a redirect_uri on the logout endpoint, its called logout. redirect_uri only exists on /authorize.

Reply:
Yes I noticed it Fred, in iOS we call both login and logout parameter as redirectUrl and in Android we call it both redirectUrl and returnToUrl.

I made this decision wondering it will be more consistent between both login and logout 🤔